adapters/appnexus/appnexus.go

```diff
@@ -87,6 +87,7 @@ type appnexusBidExtAppnexus struct {
 	BrandId       int                    `json:"brand_id"`
 	BrandCategory int                    `json:"brand_category_id"`
 	CreativeInfo  appnexusBidExtCreative `json:"creative_info"`
+	DealPriority  int                    `json:"deal_priority"`
```

Comment:
To be clear, is this just modeling the response from AppNexus? I'm really asking if an int is an appropriate type or if it should be an int pointer or if an omit empty attribute should be added to the field.

Reply:
I think here the int is the appropriate type since it will initialize to 0 from the `TypedBid` struct. A value of 0 here would be considered a bid without a deal.

Reply:
Ok, cool.